packages/components/src/components/SquiggleItem.tsx

```diff
@@ -243,8 +243,20 @@ export const SquiggleItem: React.FC<SquiggleItemProps> = ({
         </VariableBox>
       );
     }
+    case "module": {
+      return (
+        <VariableBox heading="Module" showTypes={showTypes}>
+          <span className="text-slate-600 font-semibold">Internal Module</span>
+        </VariableBox>
+      );
+    }
     default: {
-      return <>Should be unreachable</>;
+      return (
+        <>
+          <span>No display for type: </span>{" "}
+          <span className="font-semibold text-slate-600">{expression.tag}</span>
+        </>
+      );
```

Comment:
It's usually better to use divs instead of fragments for stuff like this.
It doesn't matter here but having `<div className="space-y-4"><MyComponent /></div>` backfire can be annoying.

Reply:
I don't follow? Can you explain a bit more?

Reply:
Sure. So, react fragments (`<>...</>`) allow react components to return multiple html nodes to be inserted into the parent component directly, without an intermediate div.

Such components become more affected by the context in which they are used.

```javascript
const A = () => (<><span>foo</span>{' '}<span>bar</span></>);

const B = () => (<div className="flex flex-row"><A /><div>another item</div></div>);

const C = () => (<div className="flex flex-column"><A /><div>another item</div></div>);
```

In this case, using `<B />` vs `<C />` will cause different relative positions of "foo" and "bar". `<A/>` is not self-contained anymore, you can mess up with its layout from the outside. That's _usually_ (but not always) undesirable.

Fragments are mostly useful for bits inside a single component (when you combine the final `return` of a large component with bits you prepared earlier in the same component). Or in cases when you really need the nodes to become immediate children of a parent component, e.g. if you want to return a list of `<tr>`s.

Reply:
Ah, thanks. The second line you wrote before about "space-y-4" confused me.  I changed it.